dvc/api/experiments.py

```diff
@@ -119,13 +116,6 @@ def exp_show(
             Give a negative value to include all first-parent commits (similar
             to `git log -n`).
             Defaults to 1.
-        hide_queued (bool, optional): hide experiments that are queued for
-            execution.
-            Defaults to `False`.
-        hide_failed (bool, optional): hide experiments that have failed.
-        sha (bool, optional): show the Git commit SHAs of the experiments
-            instead of branch, tag, or experiment names.
-            Defaults to `False`.
         param_deps (bool, optional): include only parameters that are stage
```

Comment:
@dberenbaum I left `param_deps` because that is a filter that you can't really apply to the result as you wouldn't have access to the info about which params are stage deps.

Side note, I am not sure why we default this option `False`

Reply:
> Side note, I am not sure why we default this option `False`

For now, I think we default to being expansive with included columns, which I think might be fine. I am guessing people would also complain with the opposite approach, and since we are increasingly moving people towards better UI than the static CLI output, it might not be a problem, but open to discussing it.